plugins/woocommerce/client/legacy/js/frontend/single-product.js

```diff
@@ -78,19 +78,6 @@ jQuery( function( $ ) {
 				// Don't do anything if gallery does not exist.
 				return;
 			}
-			setTimeout( function () {
-				var $activeElement = $( document.activeElement );
-				var sliderKeyupBlockers = [ '.stars', '.tabs', '.wc-tabs'];
-				var $closestBlocker = $activeElement.closest( sliderKeyupBlockers.join( ', ' ) );
-		
-				if ( $closestBlocker.length ) {
-					// Prevent keyup events from being triggered on the flexslider when the focus is on the stars or tabs.
-					productGalleryElement.data('flexslider').animating = true;
-					return;
-				}
-		
-				productGalleryElement.data('flexslider').animating = false;
-			}, 0);
```

Comment:
@gigitux By removing this piece of code, the gallery keeps sliding even if the user uses the keyboard to navigate through the tabs.
I explained more about my findings in [this comment](https://github.com/woocommerce/woocommerce/pull/53158/files#r2049331268).

Reply:
I could reproduce this issue @Manussakis raised (and similar related to stars vs images) but ONLY on mobile view where there's no keyboard and arrows to control the UI. I couldn't reproduce the issue on desktop view hence I think it's acceptable 🤔